keps/sig-node/2681-pod-host-ip/kep.yaml

```diff
@@ -0,0 +1,49 @@
+title: Field status.hostIPs added for Pod
+kep-number: 2681
+authors:
+  - "@wzshiming"
+owning-sig: sig-node
+participating-sigs:
+  - sig-network
+status: provisional
```

Comment:
This needs to be implementable if you want to target 1.23.

Reply:
Thanks, updated